src/helper.ts

```diff
@@ -486,19 +509,27 @@ export class Helper {
 
         for (let i = groups.length - 1; i >= 0; i--) {
             const group = groups[i];
+            const cliKey = NodeCliHelper.getCliKey(group, cliKeyMissing)
+            paths.push(`['${cliKey}']`);
             if (enumGroup) {
                 action({
-                    operationGroupCliKey: NodeCliHelper.getCliKey(group, cliKeyMissing),
+                    operationGroupCliKey: cliKey,
                     parent: groups,
                     target: group,
                     targetIndex: i,
+                    nodePath: Helper.joinNodePath(paths)
                 });
             }
-            Helper.enumrateOperations(group, action, flag);
+            
+            paths.push('operations');
+            Helper.enumrateOperations(group, paths, action, flag);
```

Comment:
>paths [](start = 45, length = 5)

doesn't matter here, but it's more efficient to pass a parsed path prefix here

Reply:
Will consider it when it's needed. Thanks